conf/eclog.yaml

```diff
@@ -14,18 +14,32 @@
 # limitations under the License.
 #
 
+
 # Configuration file for eclog
 # This configuration file can be specified by a command line argument with running eclog, or placed in either the
 # working directory or together with the binary log files.
 
-# Format of the audit record printed by eclog. Fields can be configured with bash-style parameter substitution.
-# Supported parameters are ${CLIENT_IP}, ${CLIENT_PORT}, ${COORDINATOR_IP}, ${USER}, ${BATCH_ID}, ${STATUS},
-# ${OPERATION}, ${OPERATION_NAKED}, ${TIMESTAMP}, and ${SUBJECT}.
-log_format: "${TIMESTAMP} -> Client=${CLIENT_IP}{?:${CLIENT_PORT}?}, User=${USER}, Status=${STATUS}, Operation=${OPERATION}"
 
-# Format of the ${TIMESTAMP} field as defined by the Java DateTimeFormatter. By default this field will be
-# unformatted and get printed as milliseconds since epoch of 1970-01-01Z.
-time_format: "yyyy-MM-dd HH:mm:ss z"
+# Set the format of the audit record printed by eclog.
+#
+# Fields can be configured with bash-style parameter substitution. Supported parameters are ${CLIENT_IP},
+# ${CLIENT_PORT}, ${COORDINATOR_IP}, ${USER}, ${BATCH_ID}, ${STATUS}, ${OPERATION}, ${OPERATION_NAKED}, ${TIMESTAMP},
+# and ${SUBJECT}.
+#
+log_format: "{?${TIMESTAMP}?}{?|${CLIENT_IP}?}{?:${CLIENT_PORT}?}{?|${COORDINATOR_IP}?}{?|${USER}?}{?|${STATUS}?}{?|${BATCH_ID}?}{?|${OPERATION}?}"
```

Comment:
Yes, it would be nice to get/see the default here... But should we then require this parameter to be configured? So that we remove the "duplicate" in the code? I guess we cannot since then we would break backwards compatibility... But it would be nice to only have the default at one place... Otherwise the is a risk it will differ (what is stated here and what is in the code)

But if we move that change into a different pr (as stated above) then the change will be more clear/isolated